db/migrate/20230403063433_add_akismet_index_to_post_custom_fields.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddAkismetIndexToPostCustomFields < ActiveRecord::Migration[7.0]
+  def change
+    add_index :post_custom_fields,
+              [:post_id],
+              name: "idx_post_custom_fields_akismet",
+              where: "name = 'AKISMET_STATE' AND value = 'needs_review'",
+              if_not_exists: true
```

Comment:
🆗

Reply:
Oh wait. Would it make sense to use a different name so the core migration doesn't drop this one?

Reply:
Migrations are ran based on the time stamp of the migration so this will run after core’s migration. I think it is fine to drop and recreate again.

Reply:
Uh but this assumes that the plugin will be updated and migrated before core's migration.

Reply:
Hmm good point. I’ll update the PR to use a new index name in this case.